app/test_main.py

```diff
@@ -1,3 +1,25 @@
 from app.main import get_human_age
 
-# write your code here
+def test_should_return_zero_if_zero_input():
+    assert get_human_age(0, 0) == [0, 0]
+
+def test_should_return_zero_if_less_than_cat_year():
+    assert get_human_age(14, 14) == [0, 0]
+
+def test_minimum_non_zero_result():
+    assert get_human_age(15, 15) == [1, 1]
+
+def test_minimum_non_zero_result():
```

Comment:
The function name `test_minimum_non_zero_result` is duplicated. Each test function should have a unique name to avoid confusion and ensure all tests are executed.